spring-integration-websocket/src/main/java/org/springframework/integration/websocket/ServerWebSocketContainer.java

```diff
@@ -91,6 +93,18 @@ public ServerWebSocketContainer setDecoratorFactories(WebSocketHandlerDecoratorF
 		return this;
 	}
 
+	/**
+	 * Configure allowed {@code Origin} header values.
+	 * @param origins the origins to allow.
+	 * @return the current ServerWebSocketContainer
+	 * @since 4.3
+	 * @see WebSocketHandlerRegistration#setAllowedOrigins(String...)
+	 */
+	public ServerWebSocketContainer setAllowedOrigins(String... origins) {
+		this.origins = origins;
```

Comment:
Well, I'd say `//NOSONAR` would be better.
Since this property is fully delegated and we have there a code like:
```
public WebSocketHandlerRegistration setAllowedOrigins(String... allowedOrigins) {
	this.allowedOrigins.clear();
	if (!ObjectUtils.isEmpty(allowedOrigins)) {
		this.allowedOrigins.addAll(Arrays.asList(allowedOrigins));
	}
	return this;
}
```
I don't see reason neither validate it here, nor safe via copy.